charts/karpenter/Chart.yaml

```diff
@@ -14,3 +14,10 @@ home: https://karpenter.sh/
 icon: https://repository-images.githubusercontent.com/278480393/dab059c8-caa1-4b55-aaa7-3d30e47a5616
 sources:
   - https://github.com/aws/karpenter/
+dependencies:
```

Comment:
Do we want to update these as part of the release PR? Or should we upgrade manually?

Reply:
Right now it's manual, but I will have PR to update that soon.